edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -829,6 +777,74 @@ where
     )
 }
 
+fn authenticate<MR>(
```

Comment:
By dissociating the PID check from the authorization policy, doesn't this allow module A to query the API by pretending to be module B?

Reply:
No. Authentication takes PID and query docker runtime to find a module that contains PID associated with it. Then Authorization checks name from url with auth_id depending on the policy for the route.